src/component/BpmnVisualization.ts

```diff
@@ -32,19 +33,22 @@ export default class BpmnVisualization {
    * @experimental subject to change, feedback welcome
    */
   readonly bpmnElementsRegistry: BpmnElementsRegistry;
+  private readonly _bpmnModelRegistry: BpmnModelRegistry;
```

Comment:
Not a big fan of underscore for private fields. IMHO it is enough to have linting and modifiers.

If we decide to go with that convention, we need to perhaps do the refactoring to convert other private fields to follow this approach.

Reply:
Agree, as decided with @csouchet, I will prepare a PR to make everything consistent.